apps/ledger-live-mobile/src/components/RootNavigator/types/BaseNavigator.ts

```diff
@@ -221,7 +221,8 @@ export type BaseNavigatorStackParamList = {
   [NavigatorName.RequestAccount]: NavigatorScreenParams<RequestAccountNavigatorParamList> & {
     onClose?: () => void;
   };
-  [NavigatorName.Exchange]: NavigatorScreenParams<ExchangeLiveAppNavigatorParamList> | undefined;
+  [NavigatorName.Card]: NavigatorScreenParams<PtxLiveAppNavigatorParamList> | undefined;
+  [NavigatorName.Exchange]: NavigatorScreenParams<PtxLiveAppNavigatorParamList> | undefined;
```

Comment:
In `apps/ledger-live-mobile/src/components/RootNavigator/types/PtxLiveAppNavigator.ts` we are doing a `?` after the name to signal it can be undefned, here we are doing `type | undefined`. Let's try to keep only one of those for codebase consistency

Reply:
Refactor the hole file with ?